libs/analysis/bbhnet/analysis/sensitivity.py

```diff
@@ -154,10 +146,11 @@ def __call__(
             mu = len(recovered_parameters) / num_injections
             variance = len(recovered_parameters) / num_injections**2
 
-        variance -= mu**2 / num_injections
-        v = mu * self.volume
-        variance *= v**2
+            variance -= mu**2 / num_injections
```

Comment:
The lines from here onward are indented an extra time

Reply:
good catch